examples/quantization_2of4_sparse_w4a16/README.md

```diff
@@ -74,7 +74,6 @@ apply(
     dataset=dataset,
     recipe=recipe,
     bf16=False,  # use full precision for training
-    output_dir=output_dir,
```

Comment:
why did we remove

Reply:
Ones with output_dir already set and removed, I'll revert